tests/bdd/ha-bhyve02/test_NAS_T0961.py

```diff
@@ -163,6 +163,7 @@ def click_on_system_dataset(driver):
 def the_system_dataset_page_should_open(driver):
     """the System Dataset page should open."""
     assert wait_on_element(driver, 5, xpaths.system_Dataset.title)
+    time.sleep(1)
```

Comment:
why is this here ?  can't you just add your extra time in the wait for line above?